result_test.go

```diff
@@ -140,3 +140,197 @@ func BenchmarkCreateVendorTree(b *testing.B) {
 	sm.Release()
 	os.RemoveAll(tmp) // comment this to leave temp dir behind for inspection
 }
+
+func TestStripVendor(t *testing.T) {
+	t.Run("vendor directory", stripVendorTestCase(fsTestCase{
+		before: filesystemState{
+			dirs: []fsPath{
+				fsPath{"package"},
+				fsPath{"package", "vendor"},
```

Comment:
Seems unnecessarily verbose to have to declare both of these, as `os.MkdirAll()` is the call that's used - shouldn't just the latter be sufficient?